### PR TITLE
Update new audio buttoon

### PR DESCRIPTION
**Pull request by isham-mohamed**

### CODE REVIEW

File: app/src/main/java/com/example/askapp/ui/MainActivity.kt [hunk 1]
```diff
@@ -34,14 +35,15 @@ import com.example.askapp.utils.Constants.colors
...
+import java.lang.Exception
 import java.util.*
 
 class MainActivity : AppCompatActivity(), IMainInterface,
     OnTouchListener {
     private lateinit var playingAnimation: com.wang.avi.AVLoadingIndicatorView
     private lateinit var toolbar: Toolbar
     private lateinit var viewText: TextView
-    private lateinit var audioButton: Button
+    private lateinit var audioButton: me.jagar.chatvoiceplayerlibrary.VoicePlayerView
```
Comment (Akshay-Kumar-S) [inline app/src/main/java/com/example/askapp/ui/MainActivity.kt:46]: import it and use change name audio button

File: app/src/main/java/com/example/askapp/ui/MainActivity.kt [hunk 2]
```diff
@@ -137,13 +138,14 @@ class MainActivity : AppCompatActivity(), IMainInterface,
             UtteranceProgressListener() {
             override fun onDone(utteranceId: String?) {
                 Log.d("TTF PLAYING", "Done Playing")
+                manageVoicePlay(true)
```
Comment (Akshay-Kumar-S) [inline app/src/main/java/com/example/askapp/ui/MainActivity.kt:141]: function name is not clear

File: app/src/main/java/com/example/askapp/ui/MainActivity.kt [hunk 3]
```diff
@@ -162,14 +164,21 @@ class MainActivity : AppCompatActivity(), IMainInterface,
                 R.id.record_button -> {
                     if (progressBar.isVisible) {
                         toastMessage("Previous Query Processing...")
-                    } else if (event != null) {
-                        if (textToSpeechEngine.isSpeaking) textToSpeechEngine.stop()
-                        if (mp.isPlaying) mp.stop()
+                        return false
+                    }
+                    if (event != null) {
+                        try {
```
Comment (Akshay-Kumar-S) [inline app/src/main/java/com/example/askapp/ui/MainActivity.kt:170]: Is exception handling needed?

File: app/src/main/java/com/example/askapp/ui/MainActivity.kt (hunk 1 above)
Comment (isham-mohamed) [reply app/src/main/java/com/example/askapp/ui/MainActivity.kt:46]: chaged to defalut type VoicePlayerView

File: app/src/main/java/com/example/askapp/ui/MainActivity.kt (hunk 2 above)
Comment (isham-mohamed) [reply app/src/main/java/com/example/askapp/ui/MainActivity.kt:141]: function to change the visibility of audioButton
no changed to **audioButtonVisibility**

File: app/src/main/java/com/example/askapp/ui/MainActivity.kt (hunk 3 above)
Comment (isham-mohamed) [reply app/src/main/java/com/example/askapp/ui/MainActivity.kt:170]: TTF does not have an exception handler for stop and one or two came up in my branch
not clear why just added to tes
should i remove it?

**Pull request merged**